build.rs

```diff
@@ -5,27 +5,17 @@ const REMOVED_TESTS: &[&str] = &[
     "./src/web/clients/api/rate-limited.md",
 ];
 
-fn main() {
-    #[cfg(feature = "test-rand")]
-    {
-        let rand_paths = vec![
-            "./src/algorithms/randomness/rand.md",
-            "./src/algorithms/randomness/rand-range.md",
-            "./src/algorithms/randomness/rand-dist.md",
-            "./src/algorithms/randomness/rand-custom.md",
-            "./src/algorithms/randomness/rand-passwd.md",
-            "./src/algorithms/randomness/rand-choose.md",
-        ];
-        skeptic::generate_doc_tests(&rand_paths[..]);
-        return;
-    }
+const REMOVED_PREFIXES: &[&str] = &["./src/algorithms/randomness/"];
```

Comment:
Went with a prefix approach here as it makes it easy to crank down the folders, e.g once algorithms/sorting is also done, this can just include `./src/algorithms/sorting`.